src/lib/Menu/MainMenuBuilder.php

```diff
@@ -181,6 +182,11 @@ private function getAdminMenuItems(): array
             );
         }
 
+        $menuItems[self::ITEM_ADMIN__STATE] = $this->createMenuItem(
```

Comment:
Not sure if there should be a permission check here or not and if so on what policy